ui/util/url.js

```diff
@@ -0,0 +1,68 @@
+const LBRY_INC_DOMAINS = ['lbry.io', 'lbry.com', 'lbry.tv', 'lbry.tech', 'lbry.fund', 'spee.ch'];
+
+exports.formatLbryUrlForWeb = uri => {
+  return uri.replace('lbry://', '/').replace(/#/g, ':');
+};
+
+exports.formatFileSystemPath = path => {
+  if (!path) {
+    return;
+  }
+
+  let webUrl = path.replace(/\\/g, '/');
+
+  if (webUrl[0] !== '/') {
+    webUrl = `/${webUrl}`;
+  }
+
+  return encodeURI(`file://${webUrl}`).replace(/[?#]/g, encodeURIComponent);
+};
+
+exports.isLBRYDomain = uri => {
+  if (!uri) {
+    return;
+  }
+
+  const myURL = new URL(uri);
+  const hostname = myURL.hostname;
+
+  for (let domain of LBRY_INC_DOMAINS) {
+    if (hostname.endsWith(domain)) {
+      return true;
+    }
+  }
+
+  return false;
+};
+
+/*
+  Function that handles page redirects
+  ex: lbry://?rewards
+  ex: open.lbry.com/?rewards
+*/
+exports.formatCustomUrl = path => {
```

Comment:
This is the only new thing I added to this file other than renaming. I don't really like the name of the function though.

Reply:
Agreed. `formatAppUrl`? `formatInAppUrl`? `formatAppPageUrl`?

Reply:
I like `formatInAppUrl`